libraries/PPP/src/PPP.h

```diff
@@ -94,6 +94,7 @@ class PPPClass : public NetworkInterface {
   esp_modem_flow_ctrl_t _flow_ctrl;
   int8_t _pin_rst;
   bool _pin_rst_act_low;
+  int16_t _pin_rst_delay;
```

Comment:
Please use `unsigned int` or `uint32_t` for the delay.

Reply:
sure, will fix.